password/password.cabal

```diff
@@ -200,23 +203,28 @@ test-suite password-tasty
   default-language:
       Haskell2010
 
-  if os(darwin)
-    cpp-options: -DIS_MAC_OS
-  else
-    build-depends: scrypt
-
   if flag(argon2)
     cpp-options:
       -DCABAL_FLAG_argon2
+
   if flag(bcrypt)
     cpp-options:
       -DCABAL_FLAG_bcrypt
+
   if flag(pbkdf2)
     cpp-options:
       -DCABAL_FLAG_pbkdf2
+
   if flag(scrypt)
     cpp-options:
       -DCABAL_FLAG_scrypt
+
+    -- The scrypt package is optionally used in tests to ensure compatibility
+    -- between password and scrypt.  However, scrypt doesn't work on OSX, so
+    -- we only add the scrypt dependency if we're not building on OSX.
```

Comment:
That's not enough. scrypt requires `x86_64` with SSE2 instruction set, so also aarch64-linux etc. aren't available.

Reply:
Yes. x86_64-darwin works fine (with a new enough CPU).

Reply:
Ah, thanks for the clarification here, I've updated the tests to only run on x86_64 in b5a96ef.